README.md

```diff
@@ -1,3 +1,12 @@
 # loopback-next-extension-starter
 
 A starter project to create extensions for loopback-next.
+
+## Overview
+
+The extension starter will show you how to create basic extensions for LoopBack Next. The different types of extensions are: 
+
+1. Providers
+2. Decorators
+3. Components
+4. Mixins
```

Comment:
We should talk about system-level extensions vs application-level extensions. The list you have is mostly for system extensions.

Application extensions can have:
- models
- controllers
- repositories

Reply:
Yep. I'll have a separate PR up shortly to work on the top level PR. I'll include this feedback there. I've reverted this file from this PR.